decidim-meetings/app/views/decidim/meetings/meetings/new.html.erb

```diff
@@ -1,25 +1,21 @@
-<div class="row">
-  <div class="columns large-3">
-    <%= link_to :back do %>
-      <%= icon "chevron-left", class: "icon--small", role: "img", "aria-hidden": true %>
-      <%= t(".back") %>
-    <% end %>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
+  <div class="text-center py-10">
+    <h1 class="h3 md:h1 decorator inline-block text-left">
+      <%= t("title", scope: "decidim.meetings.meetings.new") %>
+    </h1>
   </div>
 
-  <div class="columns large-6">
-    <h2 class="section-heading"><%= t(".title") %></h2>
-    <div class="card">
-      <div class="card__content">
-        <%= decidim_form_for(@form, html: { class: "meetings_form new_meeting" }) do |f| %>
-          <%= render partial: "form", object: f %>
-
-          <div class="actions">
-            <%= f.submit t(".create"), class: "button expanded", data: { disable: true } %>
-          </div>
-        <% end %>
-      </div>
+  <%= decidim_form_for(@form, html: { class: "meetings_form new_meeting" }) do |f| %>
+    <div class="form__wrapper">
```

Comment:
The start and end dates are not displaying the calendar picker when I focus on these fields:
![No calendar picker](https://user-images.githubusercontent.com/864340/209188741-e8756fc8-c936-4f87-84f8-24a1dd777a44.png)

Reply:
This was addressed in https://github.com/decidim/decidim/issues/9760. I'll move it to a separate issue as well

Reply:
https://github.com/decidim/decidim/issues/10192